tests/test_63_ecp.py

```diff
@@ -92,7 +92,7 @@ def test_complete_flow():
                                                 entity_id=sp_entity_id)
 
         resp = idp.create_ecp_authn_request_response(
-            destination, {"eduPersonEntitlement": "Short stop",
+            destination,{"eduPersonEntitlement": "Short stop",
```

Comment:
Could we restore this whitespace? Formatting is already ugly in the source files, lets try to fix it by each new commit.